### PR TITLE
Prepare branch for 6.0 work

### PR DESCRIPTION
**Pull request by ashfordneil**
Related Ticket: Get a branch ready for us to have somewhere to put our 6.0 work

Description of Changes:
* Added a placeholder release to each changelog, so we can tell 5.x work apart from 6.0 work and avoid git conflicts
* Update `build.properties` so that Jenkins knows which branch to look at when doing merges and tests
* Updated `modules/tinymce/package.json` to the new version

Pre-checks:
* [x] ~Changelog entry added~
* [x] ~Tests have been added (if applicable)~
* [x] ~Branch prefixed with `feature/` for new features (if applicable)~

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/acid/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## 4.0.1 - TBD
+
+### Added
```
Comment (lnewson) [inline modules/acid/CHANGELOG.md:11]: Why was this included?

**Review by lnewson — APPROVED** [review]

File: modules/acid/CHANGELOG.md (hunk 1 above)
Comment (ashfordneil) [reply modules/acid/CHANGELOG.md]: By mistake. Fixed now

**Review by metricjs — APPROVED** [review]
